tools/util/cassandra_stress.py

```diff
@@ -0,0 +1,52 @@
+import subprocess
+import os
+
+
+def prepare_args(command, workload_size, node_ip, keyspace="ks", threads=1):
+    return [command, "no-warmup",
+            f"n={workload_size}", 
+            "-node", node_ip, 
+            "-rate", f"threads={threads}", 
+            "-schema", f"keyspace={keyspace}"]
+
+
+class CassandraStress:
+    def __init__(self, stress_cmd):
+        self.stress_cmd = stress_cmd
+
+
+    def run(self, command, workload_size, node_ip, keyspace, threads):
+        args = [self.stress_cmd]
+        args += prepare_args(command, workload_size, node_ip, keyspace, threads)
+        subprocess.run(args=args, check=True)
+
+
+def cassandra_stress(cassandra_version="3.11.16"):
+    # Set download directory to script's directory.
+    download_dir = os.path.realpath(os.path.dirname(__file__))
+    
+    cassandra_dir = f"apache-cassandra-{cassandra_version}"
+    cassandra_tar = f"{cassandra_dir}-bin.tar.gz"
+
+    if os.path.exists(f"{download_dir}/{cassandra_dir}"):
+        # Cassandra already fetched.
+        print(f"Cassandra {cassandra_version} already installed. Skipping the download phase.")
+    else:
+        # Fetch cassandra.
+        print(f"Fetching cassandra {cassandra_version} to {download_dir}")
+        # See https://www.apache.org/dyn/closer.lua/cassandra/3.11.16/apache-cassandra-3.11.16-bin.tar.gz
+        cassandra_url = f"https://dlcdn.apache.org/cassandra/{cassandra_version}/{cassandra_tar}"
+        subprocess.run(args=["wget", "-P", download_dir, "-N", "--no-verbose", cassandra_url], check=True)
+
+        # Extract cassandra
+        print(f"Extracting cassandra {cassandra_version}")
+        subprocess.run(args=["tar", "-xzf", f"{download_dir}/{cassandra_tar}", "--directory", download_dir], check=True)
+        print(f"Extracted cassandra to {download_dir}/{cassandra_dir}")
+
+    return CassandraStress(stress_cmd=f"{download_dir}/{cassandra_dir}/tools/bin/cassandra-stress")
+
+
+def run_cql_stress(command, workload_size, node_ip, keyspace, threads):
```

Comment:
It's a little inconsistent that we have `CassandraStress` object but no `CqlStress` object. What about introducing one?